Makefile

```diff
@@ -61,7 +61,15 @@ endif
 
 DATE    := $(shell date -u '+%Y-%m-%d %H:%M UTC')
 LDFLAGS := -ldflags='-w -X "main.Version=$(VERSION)" -X "main.BuildTime=$(DATE)"'
-GOFLAGS := CGO_ENABLED=0
+GOFLAGS := -v
+
+ifeq (,$(GOFLAGS))
+	ifeq (,$(findstring CGO_ENABLED=0,$(GO_ENVS)))
+		GO_ENVS := $(GO_ENVS) CGO_ENABLED=1
```

Comment:
It's weird having `GOFLAGS` and `GO_ENVS` instead of `GO_FLAGS`/`GOENVS`

Reply:
All of the `go` commmand [environment variables](https://pkg.go.dev/cmd/go#hdr-Environment_variables) are `GO*`, and I didn't want to clash with that namespace at all. So I added `GO_ENVS` instead of `GOENVS`.

And I'm using `GOFLAGS` to refer to the official Go environment variable of that name.